Add basic file structure to the app

maayami opened the pull request:
Fix #3 
This PR adds the following file ->
- main.py
- Procfile
- runtime.txt
- requirements.txt
- example.env
- .gitignore

rajitha1998 commented on .gitignore:
```diff
@@ -0,0 +1,6 @@
+__pycache__/
+*.py
+!main.py
+.env
```
Keep a new line at the end of the file.

maayami replied:
Done

rajitha1998 commented on Procfile:
```diff
@@ -0,0 +1 @@
+worker: python main.py
```
Keep a new line at the end of the file.

maayami replied:
Updated in the new commit.

rajitha1998 commented on main.py:
```diff
@@ -0,0 +1,79 @@
+from gitterpy.client import GitterClient
```
Please add comments for readability.

maayami replied:
Done

maayami commented:
@rajitha1998 Sir, I have added a new line at the end of every file but I think it's not visible in the Github changes.
Also, I have added comments in the main.py file to improve it's readability.
